pyproject.toml

```diff
@@ -27,7 +27,7 @@ pydantic = "^2.7.4"
 qi-compute-api-client = {git = "https://github.com/QuTech-Delft/compute-api-client.git", rev = "26857ef229de7e2010e9b0915ef468b66d437598"}
 qxelarator = {version = "^0.7.1", optional = true}
 pydantic-settings = "^2.3.3"
-qiskit = "1.1.0"
+qiskit = "1.0.2"
```

Comment:
What is the reason for downgrading here?

Reply:
Ah alright 👍 
Agree that this should be picked up seperately lets downgrade for now